apps/test-suite/screens/TestScreen.js

```diff
@@ -56,11 +56,9 @@ export default class TestScreen extends React.Component {
 
     await Promise.all(
       modules.map(m =>
-        jasmine.describe(m.name, () => {
-          m.test(jasmine, {
-            setPortalChild: this.setPortalChild,
-            cleanupPortal: this.cleanupPortal,
-          });
+        m.test(jasmine, {
```

Comment:
Could you not just bubble the promise up?

Reply:
I think It's impossible in this case. 
Correct me if I'm wrong.